f2/src/forms/EvidenceInfoForm.js

```diff
@@ -176,6 +173,21 @@ export const EvidenceInfoForm = (props) => {
                       <Trans id="evidencePage.addFileButton" />
                     </FileUpload>
                   </Box>
+                  <Text fontSize="md" as="span" fontWeight="bold">
+                    <Trans id="evidencePage.supportedFiles" />
+                  </Text>
+                  <br />
+                  <Ul>
+                    <Li style={{ fontSize: 16 }}>
```

Comment:
instead of creating inline styles, you can use the fontSize and mb props! This will bring list items closer together.
```jsx
<Li fontSize="md" mb={2}>
    <Trans id="evidencePage.fileTypes2" />
</Li>
```

Reply:
fixed